### PR TITLE
[WIP] target ar71xx-mikrotik: add missing OpenWrt image types

### PR DESCRIPTION
**Pull request by txt-file**
Add the missing OpenWrt image types according to https://openwrt.org/toh/mikrotik/common#downloading_openwrt_images

sorted alphanumeric, compile tested

### CODE REVIEW

**Comment by rotanid** [comment]
@txt-file you added the x86 squashfs change to this PR, which is very wrong. please leave this PR only about mikrotik.

**Comment by txt-file** [comment]
true. I commited to the wrong branch. Removed the commit.

File: targets/ar71xx-mikrotik [hunk 1]
```diff
@@ -4,8 +4,17 @@ config 'CONFIG_GLUON_SPECIALIZE_KERNEL=y'
 # ath5k cards are commonly used with Mikrotik hardware
 packages 'kmod-ath5k'
```
Comment (mweinelt) [inline targets/ar71xx-mikrotik]: This is certainly not the case for 802.11ac compatible devices anymore. Can you speak to that?

Comment (txt-file) [reply targets/ar71xx-mikrotik]: This i probably true for older RouterBoards which had miniPCI connectors. As we probably won't support these (old) devices I can remove the line. According to [wireless.wiki.kernel.org](https://wireless.wiki.kernel.org/en/users/drivers/atheros) ath5k only supports a/b/g. I'll remove the line in my next version of the patch.

**Comment by txt-file** [comment]
I'lll prepare a new patch with added devices and broken/untested flags.

**Comment by mweinelt** [comment]
I just realised what you meant by "added devices". I don't think it's wise to track all possible devices as aliases below without having a test protocol. Nobody is going to maintain that alias list anyway.

Please, just stick to the different image types and reasonable kmod/firmware packages.

**Comment by txt-file** [comment]
I just removed kmod-ath5k. Any further changes?

Should I add devices / aliases or not?

**Comment by mweinelt** [comment]
The device aliases would imply someone took a deeper look at the individual device, as that is not the case I'd rather have them left out.

File: targets/ar71xx-mikrotik (hunk 1 above)
Comment (txt-file) [reply targets/ar71xx-mikrotik]: https://github.com/freifunk-gluon/gluon/pull/1620/commits/78386e79c99e5a7b6525a54080afe316ec9061de removes 'kmod-ath5k'

**Comment by txt-file** [comment]
alright

**Comment by mweinelt** [comment]
Replace "target ar71xx-mikrotik" with just "ar71xx-mikrotik" and then I think we're ready to merge.

**Comment by mweinelt** [comment]
Applied with the commit message changed. Thanks!

**Pull request closed without merging**